contracts/V1/StateChannelDiamondProxy/DisputeFraudProofFacet.sol

```diff
@@ -8,10 +8,8 @@ import "../types/DisputeFraudProofTypes.sol";
 import "./utils/DisputeUtils.sol";
 
 contract DisputeFraudProofFacet is StateChannelCommon {
-    mapping(
-        DisputeFraudProofType
-            => function(bytes memory encodedFraudProof, Dispute memory dispute) internal returns (address)
-    ) private proofHandlers;
+    mapping(DisputeFraudProofType => function(bytes memory, Dispute memory) internal returns (address)) private
```

Comment:
isn't it better to have `bytest memmory encodedFraudProof` than `bytes memory` which doesn't mean anything

Reply:
Naming function type parameters is deprecated.solidity(6162)